sbndqm/DatabaseStorage/Connect.h

```diff
@@ -6,9 +6,9 @@ std::string fRedisHostname;
 std::string fRedisPassword;    
 
 void RedisConnect{
-  fRedisHostname(p.get<std::string>("RedisHostname","icarus-db02")),
+  fRedisHostname(p.get<std::string>("RedisHostname","sbnd-db01")),
   fRedisPort(p.get<int>("RedisPort",6379)),
-  fRedisPassword(p.get<std::string>("RedisPassword","icarus"))
```

Comment:
Is this `Connect.h` actually used? I can't find any active `#include path/to/Connect.h` anywhere...
There is no issue in changing these defaults, but I would like to make sure we are not relying on them somewhere else and we have the correct settings in the fcl file.